src/css/InventoryCat.css

```diff
@@ -5,19 +5,32 @@
     padding: 1em 0;
 }
 
-.Category-display ul li {
+.Category {
     list-style-type: none;
     text-align: center;
     padding: .5em;
     color: #fff;
     font-family: 'Roboto Condensed', sans-serif;
     background: #231f20;
     margin: 0;
+    text-transform: uppercase;
+    font-weight: bold;
+    font-size: 1.1em;
+    letter-spacing: .07em;
 }
 
 .liquorProd {
```

Comment:
Be sure that you are being consistent in your naming conventions. I'm seeing some classes using camel case while other classes are using kebab. Setting a styleguide and sticking with it is a team is really [important](https://cssguidelin.es/)